compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/emit/TypeEmitter.java

```diff
@@ -233,11 +234,11 @@ private static String emitBTupleType(BTupleType bType, int tabs) {
             return bType.toString();
         }
         StringBuilder tupleStr = new StringBuilder("(");
-        int length = bType.tupleTypes.size();
+        int length = bType.memberTypes.size();
         int i = 0;
-        for (BType mType : bType.tupleTypes) {
-            if (mType != null) {
-                tupleStr.append(emitTypeRef(mType, tabs));
+        for (BTupleMember tupleMember : bType.memberTypes) {
```

Comment:
`org.wso2.ballerinalang.compiler.semantics.model.types.BTupleType#memberTypes` should be `members`, not `memberTypes`? Please fix.